response.go

```diff
@@ -0,0 +1,21 @@
+package common
+
+// NewResponse will return a new Response
+func NewResponse(statusCode int, contentType string, value interface{}) *Response {
```

Comment:
@itsmontoya is this going to handle callbacks for NewJSONPResponse?

Reply:
Great call, added it!